### PR TITLE
Display log

### LINKED ISSUE #25757: The Logger() setup by utils.display uses an unconventional name
##### SUMMARY

The logging.Logger() class setup by ansible.utils.display() uses an unconventional name  ('p=$PID u=$USER | ')

``` python
     logger = logging.getLogger("p=%s u=%s | " % (mypid, user))
```

Normal convention is to use the module name for the Logger() name, ie

``` python
    logger = logging.getLogger(__name__)
```

https://docs.python.org/2/howto/logging.html#advanced-logging-tutorial

> A good convention to use when naming loggers is to use a module-level logger, in each module which uses logging, named as follows:
> 
> logger = logging.getLogger(__name__)
> 
> This means that logger names track the package/module hierarchy, and it’s intuitively obvious where events are logged just from the logger name.

This results in the Logger() used for display being hard to reference from other code or from logging configuration since the logger name is dynamic. If a different log handler is added to that logger (for ex, a syslog handler), then the log events are generated with unusual and unpredictable names.

##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

lib/ansible/utils/display.py

##### ANSIBLE VERSION

```
ansible 2.4.0 (devel 44687bb917) last updated 2017/06/15 13:12:13 (GMT -400)
  config file = None
  configured module search path = [u'/home/adrian/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/adrian/src/ansible/lib/ansible
  executable location = /home/adrian/src/ansible/bin/ansible
  python version = 2.7.13 (default, May 10 2017, 20:04:28) [GCC 6.3.1 20161221 (Red Hat 6.3.1-1)]


```

##### CONFIGURATION

```
[defaults]
log_path = /path/to/some/log/file.log
```
##### ACTUAL RESULTS

```

```

### PR DESCRIPTION
**Pull request by alikins**
##### SUMMARY

Address some of the display log issues, at least stuff that can be addressed without more
significant changes.

Not as good or as useful as a conventional use of logging, but it's an improvement.

Fixes #25757, #25758, #25759, #25761 

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

lib/ansible/utils/display.py
##### ANSIBLE VERSION

```
ansible 2.4.0 (display_log cc5bec3df2) last updated 2017/06/15 14:45:29 (GMT -400)
  config file = None
  configured module search path = [u'/home/adrian/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/adrian/src/ansible/lib/ansible
  executable location = /home/adrian/src/ansible/bin/ansible
  python version = 2.7.13 (default, May 10 2017, 20:04:28) [GCC 6.3.1 20161221 (Red Hat 6.3.1-1)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/utils/display.py [hunk 1]
```diff
@@ -47,17 +47,45 @@
...
+VVVV = VVV - 1
+VVVVV = VVVV - 1
+VVVVVV = VVVVV - 1
+
+logging.V = V
+logging.VV = VV
+logging.VVV = VVV
+logging.VVVV = VVVV
+logging.VVVVV = VVVVV
+logging.VVVVVV = VVVVV
```
Comment (tumbl3w33d) [inline lib/ansible/utils/display.py:65]: This one looks wrong

**Comment by karniemi** [comment]
Would this PR provide the original feature requested in #13072 ? Is there a doc for it? (The feature was claimed to have been implemented in #13593 ... but that does not seem user-friendly in any way...or maybe I'm not able to read between the lines.)

**Comment by abadger** [comment]
I now have logging on my radar.  Will track progress here: https://github.com/ansible/ansible/issues/49657

**Comment by bcoca** [comment]
closed via #56311

**Pull request closed without merging**